source/Conversation.h

```diff
@@ -75,9 +74,9 @@ class Conversation {
 	bool IsChoice(int node) const;
 	int Choices(int node) const;
 	bool IsBranch(int node) const;
-	bool IsApply(int node) const;
+	bool IsAction(int node) const;
 	const ConditionSet &Branch(int node) const;
```

Comment:
A slight nitpick, but if Apply is now GetAction then I'd think Branch would now be GetBranch.

Reply:
You're not wrong, getters generally shouldn't have an imperative tone that makes them read like a non-getter. I tried to not stray too far into things that weren't modified by the other PR so I didn't do things like that, adding `const noexcept` to all the getters, etc.